pgjdbc/src/main/java/org/postgresql/jdbc/PgArray.java

```diff
@@ -793,6 +793,7 @@ private Object buildArray(PgArrayList input, int index, int count) throws SQLExc
 
       throw org.postgresql.Driver.notImplemented(this.getClass(), "getArrayImpl(long,int,Map)");
     }
+    System.out.println("testing get object data in pgarray-->" + ret);
```

Comment:
Please remove any System.out.println use logging instead

Reply:
It is the outdated version and I have already removed System.out.println. Could you please check overall commit? There are only 2 file I have changed, PgConnection and PgResultSet.